modules/genomic_browser/jsx/tabs_content/cnv.js

```diff
@@ -0,0 +1,160 @@
+    return fetch(window.location.origin + '/genomic_browser/CvnBrowser',
+      {credentials: 'same-origin'}
+    )
+      .then((resp) => resp.json())
+      .then((json) => {
+        // console.log(json);
+        const data = {
+          fieldOptions: json.fieldOptions,
+          Data: json.data.map((e) => Object.values(e)),
+          subprojects: json.subprojects,
+        };
+        this.setState({data});
+      }).catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {array} rowData - array of cell contents for a specific row
+   * @param {array} rowHeaders - array of table headers (column names)
+   *
+   * @return {*} a formatted table cell for a given column
+   */
+  formatColumn(column, cell, rowData, rowHeaders) {
+    let reactElement = null;
+    if (true) {
+      switch (column) {
+        case 'PSCID':
+          const url = window.location.origin + '/' + rowData.DCCID + '/';
+          reactElement = (
+            <td><a href={url}>{rowData.PSCID}</a></td>
+          );
+          break;
+        case 'Subproject':
+          reactElement = (
+            <td>{this.state.data.subprojects[parseInt(cell)]}</td>
+          );
+          break;
+        default:
+          reactElement = (
+            <td>{cell}</td>
+          );
+          break;
+      }
+    }
+    return reactElement;
+  }
+
+  /**
+   * @return {DOMRect}
+   */
+  render() {
+    // Waiting for async data to load.
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+
+    let fieldOptions = this.state.data.fieldOptions;
+    let fields = [];
+    for (let field in fieldOptions) {
+      if (fieldOptions.hasOwnProperty(field)) {
+        if (fieldOptions[field].name === 'dob' ||
+          fieldOptions[field].name === 'Array_Report' ||
+          fieldOptions[field].name === 'Validation_Method' ||
+          fieldOptions[field].name === 'Platform' ||
+          fieldOptions[field].name === 'Markers' ||
+          fieldOptions[field].name === 'Gene_Name' ||
+          fieldOptions[field].name === 'Gene_Symbol' ||
+          fieldOptions[field].name === 'Location' ||
+          fieldOptions[field].name === 'End_Loc' ||
+          fieldOptions[field].name === 'Start_Loc'
+        ) {
+          fields.push({
+            label: fieldOptions[field].label,
+            show: fieldOptions[field].hidden,
+          });
+        } else {
+          fields.push({
+            label: fieldOptions[field].label,
+            show: fieldOptions[field].hidden,
+            filter: {
+              name: fieldOptions[field].name,
+              type: fieldOptions[field].type,
+              options: fieldOptions[field].options,
+            },
+          });
+        }
+      }
+    }
+
+    return (
+      <div>
```

Comment:
`<div>`'s are unnecessary in this case because `FilterableDataTable` is self-contained.